### ISSUE TITLE
Android build fails for 5.2.0 with RN 0.68.2

### ISSUE THREAD

**Issue by TorKip** [TheWidlarzGroup/react-native-video#2765]
### Current behavior
Describe what happens when you encounter this issue.
Android build fails with newer react native.

Following error occurs

Could not determine the dependencies of task ':react-native-video:compileReleaseAidl'.
> Could not resolve all task dependencies for configuration ':react-native-video:releaseCompileClasspath'.
   > Could not find com.google.android.exoplayer:exoplayer:2.13.2.
     Required by:
         project :react-native-video
   > Could not find com.google.android.exoplayer:extension-okhttp:2.13.2.
     Required by:
         project :react-native-video

### Reproduction steps
A 1, 2, 3, etc. list of what's needed to see the issue happen.

Using react-native 0.68.2
run `yarn add react-native-video` (5.2.0)
run `npx react-native run-android`

### Expected behavior
Describe what you wanted to happen

Build succeeds

### Platform
Which player are you experiencing the problem on:
* Android

### Video sample
If possible, include a link to the video that has the problem that can be streamed or downloaded from.

Not appliccable

### Solution

Update shaka-player to 3.2.1
update com.google.android.exoplayer:exoplayer to 2.13.3
update com.google.android.exoplayer:extension-okhttp to 2.13.3

**Comment by TorKip** [comment]
I know version 6 is on its way, which will likely make this issue obsolete, but I thought i would suggest this change, as it would be nice to continue using react-native-video also before version 6 is official.

Let me know if this is wanted and i will make a PR fixing the issue

**Comment by mizutani256** [comment]
> Update shaka-player to 3.2.1
> update com.google.android.exoplayer:exoplayer to 2.13.3
> update com.google.android.exoplayer:extension-okhttp to 2.13.3

I'm not overly familiar with node/yarn (or react-native, for that matter), how can I apply this solution?

**Comment by TorKip** [comment]
What I ended up doing was create a patch that yarn can apply automatically, so I would not need to manually edit these files every time I run yarn.
I suggest following this guide: https://dev.to/ajmal_hasan/make-changes-to-nodemodules-files-with-patch-package-updated-52af
TL;DR: install `patch-package` and `postinstall`, manually edit `node_modules/react-native-video/package.json` and `node_modules/react-native-video/android-exoplayer/build.gradle` with the changes mentioned above, create patch.

**Comment by mizutani256** [comment]
> What I ended up doing was create a patch that yarn can apply automatically, so I would not need to manually edit these files every time I run yarn. I suggest following this guide: https://dev.to/ajmal_hasan/make-changes-to-nodemodules-files-with-patch-package-updated-52af TL;DR: install `patch-package` and `postinstall`, manually edit `node_modules/react-native-video/package.json` and `node_modules/react-native-video/android-exoplayer/build.gradle` with the changes mentioned above, create patch.

Great, I'll check it out! Thanks!

**Comment by freeboub** [comment]
I just backported some known fixes on this branch: https://github.com/react-native-video/react-native-video/pull/2795/files
Can you have a quick test with it please ?

**Comment by freeboub** [comment]
release 5.2.1 has been published, let's close this issue
